pkg/resource/db_cluster/custom_update.go

```diff
@@ -542,6 +542,7 @@ func (rm *resourceManager) newCustomUpdateRequestPayload(
 	res := &svcsdk.ModifyDBClusterInput{}
 
 	res.SetApplyImmediately(true)
+	res.SetAllowMajorVersionUpgrade(true)
```

Comment:
Updating `AllowMajorVersionUpgrade` is only allowed for Amazon Aurora DB Clusters. I supposed the API will reject all the update operations for non Aurora DBclusters

Reply:
I think the message  `Valid for: Aurora DB clusters only` was kind of outdated. aurora cluster is the first cluster concept introduced in rds, later when global db cluster and multi-az cluster comes in, the doc is not updated.
if you see modify global cluster, this flag is also available. https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_ModifyGlobalCluster.html